code/core/src/telemetry/run-telemetry-operation.ts

```diff
@@ -0,0 +1,22 @@
+import { createFileSystemCache, resolvePathInStorybookCache } from '../common';
+
+const cache = createFileSystemCache({
+  basePath: resolvePathInStorybookCache('telemetry'),
+  ns: 'storybook',
+  ttl: 24 * 60 * 60 * 1000, // 24h
```

Comment:
Do we want to lower the TTL?

Reply:
🤷 I figured we'd thought about it before :)